README.md

```diff
@@ -97,7 +97,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-You can find a live demo in [HERE](https://github.com/misikira).
+You can find a live demo in [HERE](https://misikira.github.io/Confference-web-app/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Well done recording the presentation video as requested by the previous code reviewer:+1: Please remember from the project requirement, your presentation video should be a maximum length of 5 minutes but currently the length of your presentation video is almost 8 minutes so I suggest your record another video showcasing your project. :pray: 
 - Remember that the video 
   - is the maximum length of 5 minutes.
   - contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
   - contains a demo of the project, showing the pages in different screen sizes.
   - contains a highlight about a piece of the code the you finds interesting or is proud of making.

- [ OPTIONAL] Why don't you write a professional project description explaining what your project is about in your readme and include the presentation video in the readme. :blush: